### PR TITLE
Configure systemd journal to forward to syslog

### PR DESCRIPTION
**Pull request by jsf9k**
This way any logs that are sent to journald will be forwarded by Amazon CloudWatch Agent to CloudWatch.

### CODE REVIEW

File: molecule/tests/test_default.py [hunk 1]
```diff
@@ -28,7 +28,16 @@ def test_files(host, f):
...
     assert host.service(service).is_enabled
+
+
+def test_systemd_journald_config(host):
+    """Test that the journald config was altered as expected."""
+    f = host.file("/etc/systemd/journald.conf")
+    assert f.exists
+    assert f.is_file
+    assert f.contains(r"^ForwardToSyslog=yes")
+    assert not f.contains(r"^ForwardToSyslog=no")
```
Comment (felddy) [inline molecule/tests/test_default.py:43]: Rule 2 The Double tap

File: molecule/ubuntu-only/prepare.yml [hunk 2]
```diff
@@ -1,4 +1,20 @@
 ---
+# The Ubuntu 16.04 Docker image we are using
+# (geerlingguy/docker-ubuntu1604-ansible:latest) needs a cache update
+# before it can install anything.
+- name: Group hosts by OS distribution and major version
+  hosts: all
+  tasks:
+    - name: Group hosts by OS distribution and major version
+      group_by:
+        key: os_{{ ansible_facts['distribution'] }}_{{ ansible_facts['distribution_major_version'] }}
```
Comment (felddy) [inline molecule/ubuntu-only/prepare.yml:10]: My, what a long line you have there. ;)

**Review by felddy — APPROVED** [review]
You made this look easy.   Nice work.

**Review by dav3r — APPROVED** [review]
Looking good! 📶

**Pull request merged**